xunit-autolabeler-v2/ast_parser/core/cli.py

```diff
@@ -36,12 +37,196 @@ def _write_output(output: List[str], output_file: str) -> None:
             print(line)
 
 
+def list_region_tags(
```

Comment:
Can we separate the logic and formatting part?
I feel like this function contains both the core logic and the data rendering code.

Maybe you'll need to have a data structure for representing the result.

Reply:
If we decoupled them, we'd have to send _all_ the result components to the data rendering code (even those that aren't rendered, as it would be the rendering code's job to discard them).

Considering that: is this still something we want to do? (If it is, I'll go ahead and change this.)

Reply:
Sorry, I don't quite understand what you're saying in the first part, but to directly answer your last question, yes.

Reply:
Refactored these into separate files; marking this as resolved.